CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Here is a template for new release sections
  - Tests for source components in D1 (#391)
  - Option `-i` for `python mvs_report.py`, `python mvs_report.py -h` for help (#407)
  - Pyppeteer package for OS X users in troubleshooting (#414)
+ - Add an enhancement to the auto-report by printing the log messages such as warnings and errors (#417)
```

Comment:
```suggestion
 - Enhance to the auto-report by printing the log messages such as warnings and errors (#417, #426)
```

@Bachibouzouk told me that changelog entries link to the PR, not the issue that they focus on. That way if something is weird you can directly jump to the specific changes. I am not sure if you have a reason to also mention #417, so I leave it to you if you want to drop or keep it.